packages/inventory/README.md

```diff
@@ -74,3 +60,75 @@ const props = {
   - Route to the service point profile view to redirect to after pressing `Cancel` button on the form to abort and exit the view or after successful submission.
   - If not provided the default is `/inventory/profile/<id>` where `id` is the id of the service point
   - If provided, the id of the service point will be added to the route name e.g if you passed `/inventory/service-point` as the `servicePointProfileURL`, the redirect will be to `/inventory/service-point/<id>`
+  -
+
+### Inventory List
+
+To add a view for inventory list:
+
```

Comment:
I find adding screenshot of the component after the description really complementary. @haseemVD What do you think?

Reply:
@kelvin-muchiri I think this is sth that could easily get stale and out of sync with the code itself just like how our documentation has been, so I do not necessarily think its something we should add here, though we could leverage something like storybooks to showcase the UI/UX instead of using static images.
cc @HaseemUlHaq

Reply:
@p-netm Ok no problem....I have no strong feelings about it either